src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -234,13 +257,21 @@ export class WorkspaceSavedObjectsClientWrapper {
       attributes: T,
       options?: SavedObjectsCreateOptions
     ) => {
-      if (isWorkspacesLikeAttributes(attributes)) {
-        await this.validateMultiWorkspacesPermissions(
-          attributes.workspaces,
+      let workspacePermitted;
+      if (options?.workspaces && options.workspaces.length > 0) {
```

Comment:
It doesn't check `options.workspaces.length > 0` in `bulkCreateWithWorkspacePermissionControl`, should it align here?